content/en/news/releases/1.16.x/announcing-1.16/change-notes/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Istio 1.16.0 Change Notes
+linktitle: 1.16.0
+subtitle: Minor Release
+description: Istio 1.16.0 change notes.
+publishdate: 2022-11-15
+release: 1.16.0
+weight: 10
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** fetching charts from URLs in `istio-operator`.
+
+## Traffic Management
+
+- **Improved** sidecar `Host` header matching to ignore port numbers by default. This can be controlled by the `SIDECAR_IGNORE_PORT_IN_HOST_MATCH` environment variable.
+
```

Comment:
@hzxuzhonghu : wouldn't it be nice to include https://github.com/istio/istio/pull/36639/ in releasenotes? Looks like it is missing at the moment?

Reply:
Yes, better to include https://github.com/istio/istio/pull/36639

Reply:
@dhawton: you could add something like "- **Updated** `meshConfig.discoverySelectors` to dynamically limit the Custom Resource Configurations (like Gateway, VirtualService, DestinationRule, Ingress etc.) if the `ENABLE_ENHANCED_RESOURCE_SCOPING` feature flag is enabled."

Reply:
We typically include issues instead of PRs, so I've added the linked issue from that PR: https://github.com/istio/istio/issues/36627.